plugins/poll/plugin.rb

```diff
@@ -171,15 +171,13 @@ class Error < StandardError
   on(:merging_users) do |source_user, target_user|
     DB.exec(<<-SQL, source_user_id: source_user.id, target_user_id: target_user.id)
       DELETE FROM poll_votes
-      WHERE poll_id IN (
-        SELECT poll_id
-        FROM poll_votes
-        WHERE user_id = :source_user_id
-        INTERSECT
-        SELECT poll_id
+      WHERE user_id = :source_user_id
```

Comment:
I am thinking out loud here:

If users A and B are merged and user B already voted, then votes of user A are deleted from the database. What happens if this is a multiple choice poll? Should we merge the options?

For example, user A voted (1, 2) and user B voted (2, 3). Merging the users should result in (1, 2, 3)? I do not think because that might cause problems some min/max requirements.

Reply:
Yeah I get you.

I spoke with JammyDodger about this as well and we reached the conclusion that keeping the vote of the target user only is good.

For your example, I think we also have to cater for the "max" votes in the multiple-type polls. So if "max" was 2 votes in the poll, we can only keep (1,2). Would be more straightforward to keep the target's vote.